bot/constants.py

```diff
@@ -143,6 +143,7 @@ class Client(NamedTuple):
     prefix = environ.get("PREFIX", ".")
     token = environ.get("BOT_TOKEN")
     debug = environ.get("BOT_DEBUG", "true").lower() == "true"
+    in_ci = environ.get("IN_CI").lower() == "true"
```

Comment:
What happens if this is unset?

Reply:
It would fail 😓

Fixed in https://github.com/python-discord/sir-lancebot/pull/1039/commits/3aa76661c68787dac9b7c9fa490f94941fa7760d

Reply:
**As a...**
Python Discord Admin

**I want to...**
  Only allow starting our community bot in CI

**Such that...**
    I have less robots that can mess with my work